src/Modules/MotdModule/Database/Repositories/MotdRepository.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using EvoSC.Common.Database.Models.Player;
+using EvoSC.Common.Database.Repository;
+using EvoSC.Common.Interfaces.Database;
+using EvoSC.Common.Interfaces.Models;
+using EvoSC.Common.Services.Attributes;
+using EvoSC.Common.Services.Models;
+using EvoSC.Modules.Official.MotdModule.Database.Models;
+using EvoSC.Modules.Official.MotdModule.Interfaces;
+using LinqToDB;
+
+namespace EvoSC.Modules.Official.MotdModule.Database.Repositories;
+
+[ExcludeFromCodeCoverage(Justification = "Database is not testable.")]
```

Comment:
We can create integration tests for database repositories, check out the repository tests under `Tests/EvoSC.Common.Tests/Database` for inspiration :)

Reply:
added MotdRepositoryTests.cs.